src/sonic-bgpcfgd/tests/data/sonic-cfggen/bgpd.conf.j2/all.conf

```diff
@@ -56,10 +61,11 @@ router bgp 55555
     network fc00::1/64
   exit-address-family
 !
-  network 10.10.10.1/24
   address-family ipv6
    network fc01::1/64
   exit-address-family
+  network 10.10.10.1/24
```

Comment:
is this intended?

Reply:
this is an artefact of order change. Also, dict insertion order would depend on dict implementation on Python 2/3.